src/Faker/Provider/ar_PS/Internet.php

```diff
@@ -17,12 +17,41 @@ class Internet extends \Faker\Provider\Internet
     protected static $tld = array(
         'biz', 'com', 'info', 'jo', 'net', 'org',
     );
-
+    
+    /**
+     * Arabic masculine given names
+     * @link    https://en.wikipedia.org/wiki/Category:Arabic_masculine_given_names
```

Comment:
there are much more names there than what you include in the formatter. Can you explain how you chose them?